app/src/main/java/com/cornellappdev/uplift/ui/screens/ProfileCreationScreen.kt

```diff
@@ -0,0 +1,227 @@
+package com.cornellappdev.uplift.ui.screens
+
+import android.widget.ImageButton
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material3.Checkbox
+import androidx.compose.material3.CheckboxColors
+import androidx.compose.material3.CheckboxDefaults
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
```

Comment:
When I open this PR on my android studio, it says that all the `material3` imports are failing. Is there some gradle stuff you didn't push for a `material3` dependency? I rebuilt and it's still happening.

![image](https://github.com/user-attachments/assets/5f0b51d9-4d1b-4021-8cac-144c60555b3a)